python_modules/libraries/dagster-components/dagster_components/cli/list.py

```diff
@@ -56,3 +57,36 @@ def _add_component_type_to_output(
         package=package,
         **component_type.get_metadata(),
     )
+
+
+@list_cli.command(name="all-components-schema")
+@click.pass_context
+def generate_code_location_components_schema(ctx: click.Context) -> None:
+    """Builds a JSON schema which ORs the schema for a component
```

Comment:
opted for this route rather than trying to assemble an ORed schema ourselves from the existing schema metadata, since this has pydantic do the heavy lifting of combining `$defs`, `$ref` remapping etc